crates/cadenza-ir/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
While I like the separation of concerns, it's going to be difficult to have the eval and IR in different crates. Eval needs to be able to emit IR directly. And IR needs to be aware of eval values and types. They're ultimately bidirectional dependencies.